.github/workflows/build.yml

```diff
@@ -123,14 +123,24 @@ jobs:
 
             - name: Configure and build.
               run : |
-                  cmake -S .    --preset=${{ matrix.compiler_family }}-${{ matrix.backend }} --warn-uninitialized -Werror=dev
+                  cmake -S .    --preset=${{ matrix.compiler_family }}-${{ matrix.backend }} --warn-uninitialized -Werror=dev -DCMAKE_INSTALL_PREFIX=/opt/test-folder-for-install
                   cmake --build --preset=${{ matrix.compiler_family }}-${{ matrix.backend }}
 
+            - name: Install.
+              run : |
+                  bash tests/install.sh ${{ matrix.compiler_family }}-${{ matrix.backend }} /opt/test-folder-for-install
+
             - name: Test.
               if  : ${{ matrix.run_tests }}
               run : |
                   ctest --preset=${{ matrix.compiler_family }}-${{ matrix.backend }}
 
+            - name: Find package against install.
+              run : |
+                  test -d include/
+                  rm -rf include/
```

Comment:
Still needed?

Reply:
It's a safety measure to ensure that only the installed files exist.